web_widget_date_interval/static/src/js/web_widget_date_interval.js

```diff
@@ -75,17 +75,22 @@ openerp.web_widget_date_interval = function (instance) {
                 }
                 return result;
             },
+            weeknumber_iso_date_from_week: function (year, week) {
+                return new Date(year, 5, 1).setWeek(week);
```

Comment:
please add a comment here that you just have to fill in some date here for the constructor, it puzzled me for a long time why it needs to be exactly this date